lib/broken_record/job.rb

```diff
@@ -58,16 +58,23 @@ def primary_key
     end
 
     def record_ids
-      records_per_group = (model_scope.count / parallelization.to_f).ceil
-      scope = model_scope.offset(records_per_group * index)
+      records_per_group = (models_with_conditions.count / parallelization.to_f).ceil
+      scope = models_with_conditions.offset(records_per_group * index)
       scope.limit(records_per_group).pluck(primary_key)
     end
 
-    def model_scope
-      default_scope = BrokenRecord::Config.default_scopes[klass] || BrokenRecord::Config.default_scopes[klass.to_s]
+    def models_with_includes
+      apply_scope(BrokenRecord::Config.model_includes)
+    end
+
+    def models_with_conditions
+      apply_scope(BrokenRecord::Config.model_conditions)
+    end
```

Comment:
Allows you to separate `includes` relation elements and `where` relation elements in any project's broken_record initializer.